### ISSUE TITLE
Strange color rendering

### ISSUE THREAD

**Issue by nicolabotta** [bordaigorl/rmview#134]
Thanks for providing rmview, it's a great tool! 

I was wandering whether color rendering is supported. I have played a bit around and I get some weird result. Writing in red with the Ballpoint pen is rendered in black with some red strokes. 

Highlighting on text works as expected but on a white background the results are quite strange, please see below. Similarly for other tools that support colors.

My device is a rM1 and I am on 2.12.1.527, the host runs Debian stable:

`Linux cirrus 5.10.0-11-amd64 #1 SMP Debian 5.10.92-1 (2022-01-18) x86_64 GNU/Linux`

My rmview configuration is

```
{
  "ssh": {
    "address": ["10.11.99.1"],
    "username": "root",
    "auth_method": "key",
    "key": "/home/nicola/.ssh/id_rsa",
    "timeout": 2
  },
  "orientation": "auto",
  "pen_size": 15,
  "pen_color": "red",
  "pen_trail": 50,
  "background_color": "white",
  "hide_pen_on_press": true
}
```

![rm-screenshot](https://user-images.githubusercontent.com/2726946/154148946-0de52fc5-fbc2-433c-b6c7-d8b1a1c14eff.png)

Best+thanks! N.

**Comment by bordaigorl** [comment]
I did a couple of tests: it seems the problem is there only when annotating pdfs. If I try on a notebook the colors render without issues.
On PDFs I can see the artifacts of your screenshot, and they disappear if you change page and go back (thus triggering a full refresh).
I suspect this is a problem with the built-in VNC server running on the tablet, but to confirm one would need to reproduce the issue on the official app.
One workaround I could code up is to provide a "refresh" menu item that requests the full screen data from scratch (my guess is that the artifacts are causing by the server not sending updates for all the affected areas but only some).

**Comment by nicolabotta** [comment]
I would be happy to try to reproduce the issue on the official app but having upgraded to Debian stable (bullseye) to install rmview I am now unable to install wine32 and run the official windows app under wine. I'll give it another try later but support for multiarchitecture appears to be broken.

**Comment by nicolabotta** [comment]
Ok, I have managed to run the official windows app under wine on another machine running Debain buster (oldstable). As it turns out, color rendering was broken under `reMarkable-2.11.0.182-win32.exe`:

![Screenshot_2022-02-16_13-12-30](https://user-images.githubusercontent.com/2726946/154263848-94e02068-0698-42c6-8f73-e17f272e1773.png)

but seems to work with `reMarkable-2.12.1.187-win32.exe`:

![Screenshot_2022-02-16_13-21-07](https://user-images.githubusercontent.com/2726946/154263992-86fb6a93-6e53-4326-9049-5ae131c37793.png)

**Comment by bordaigorl** [comment]
🤔 that's weird.
If I had to guess, it is possible that when you tried with 2.12.1.187 a full refresh was triggered (which would have solved the issue with 2.11.0.182 too).
Could you do one more test using 2.12.1.187 but starting with a PDF from scratch and draw in color while screensharing?

**Comment by nicolabotta** [comment]
Done: you are right, 2.12.1.187 needs a refresh except when highlighting text which is immediatly rendered in color, sorry for the noise!

The only problem I am left with is that rmview needs 5-6 seconds to turn a page on my laptop (Thinkpad x230), the windows app under wine takes about 1 second, more or less like the rM. 

This is a bit strange because writing in rmview is at least as fast as writing with the official windows app under wine!
